packages/v4/src/content/design-guidelines/components/alert/alert.md

```diff
@@ -4,123 +4,134 @@ section: components
+Inline alerts are commonly used in the following situations:
+- Informing the user that the information they entered into a form field isn’t valid
+- Informing the user of something that needs their attention
+- Informing the user that they need to set something up
+- Informing the user that a feature is temporarily unavailable
 
-#### When to use
-Use an inline alert when you want to return feedback as the result of a user action within a page while allowing the user to continue working. Common uses for inline alerts include form validation or warning messages that should not interrupt the user’s flow.
+## Using toast alerts
+Use toast alerts to communicate an update, confirmation, or other activity to the user without blocking their workflow. For example, you can use a toast alert to inform the user that their message was sent. You can include links in a toast alert for a user to take action on the event that surfaced the alert, but the links should only be a shortcut for the user. A toast alert should never be used as the only means for taking actions on an alert event.
 
-An inline alert should never block the user from interacting with other content on a page. For informational alerts, the user should be able to dismiss the alert by clicking the Close button on the alert. For error conditions, we recommend that the alert is dismissed only when the condition raising it has been cleared.
+Toast alerts stack in the upper-right corner of a page. To properly position and stack a set of alerts, use an [alert group](/components/alert-group) component. When alerts stack, the most recent alert appears at the top, and others move down as more alerts come in. When alerts are deleted, the remaining alerts move up.
 
-In addition to static message content and links, interactive elements may optionally be included in the body of an inline alert to ask the user for further input or feedback.
+![alert group](./img/toast-notification-multiple.png)
 
-<img src="./img/inline-interactive-alert.png" alt="interactive inline alert" width="439"/>
+Toast alerts should always have a close button so that the user can dismiss them. If you include an automatic timeout for your toast alert, 8 seconds is recommended. Toast alerts should not disappear while the user is hovering over them.
 
-#### When not to use
-Do not use inline alerts to reflect the status of an asynchronous operation. In this case, a Toast Alert should be used.
+<img src="./img/Toast-alerts-desktop.png" alt="toast alerts in a page" />
 
-#### Alternative solutions
-If the user must take action to address a situation before proceeding consider using a [modal](/components/modal/design-guidelines). Inline alerts are preferred to a modal when the problem can be corrected on the current page, however.
+Toast alerts are commonly used in the following situations:
+- Confirming that a message sent
+- Confirming that an update was made
+- Informing the user that their action was completely successfully
```

Comment:
Should say "completed" instead of "completely" haha

Reply:
Lol, my bad!